packages/client-app/src/components/DataTable/DataTable.vue

```diff
@@ -6,7 +6,7 @@ defineProps<{
 }>()
 </script>
 <template>
-  <div
+  <form
```

Comment:
setting that back to comply with good practice and fix error log in chrome, any reason to have removed it @cameronrohani?

Reply:
the table doesn't show up for form data or other body elements when it's a form :)

Reply:
oh make sense no form within table, updated 🙏